stanford-sw/sw_index.properties

```diff
@@ -177,6 +177,8 @@ building_facet = custom, getBuildings, library_map.properties
 item_display = customDeleteRecordIfFieldEmpty, getItemDisplay
 
 mhld_display = custom, getMhldDisplay
+bookplates_display = custom, getBookplatesDisplay
+fund_facet = custom, getFundFacet

```

Comment:
@lmcglohon perhaps I'm letting my java naiveté show but shouldn't `getBookplatesDisplay` and `getFundFacet` be defined somewhere since they are custom functions?